docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,229 @@
+
+## Interacting with the spawned network
+
+### Connecting to the nodes
+
+After the network is launched, you can interact with it using the [Polkadot.js App](https://polkadot.js.org/apps/){target=_blank}. To do so, open your browser and use the provided links listed by the output as 'direct link'. For instance, in this particular case, as the ports may vary from spawning to spawning, to interact with the `alice` node, open [https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer](https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer){target=_blank} as it is the link provided in the output for the `alice` node. Moreover, you can also do this for the `bob` and `collator01` nodes.
+
+If you want to interact with the nodes more programmatically, you can also use the [Polkadot.js API](https://polkadot.js.org/api/){target=_blank}. For example, the following code snippet shows how to connect to the `alice` node using the Polkadot.js API and log some information about the chain and node:
+
+```typescript
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+async function main() {
+  const wsProvider = new WsProvider('ws://127.0.0.1:55308');
+    const api = await ApiPromise.create({ provider: wsProvider });
+
+    // Retrieve the chain & node information via rpc calls
+    const [chain, nodeName, nodeVersion] = await Promise.all([
+        api.rpc.system.chain(),
+        api.rpc.system.name(),
+        api.rpc.system.version()
+    ]);
+
+    console.log(`You are connected to chain ${chain} using ${nodeName} v${nodeVersion}`);
+}
+
+main().catch(console.error).finally(() => process.exit());
+```
+
+Either way allows you to interact easily with the network and its nodes.
+
+### Checking metrics
+
+You can also check the metrics of the nodes by accessing the provided links listed by the output as 'Prometheus Link'. [Prometheus](https://prometheus.io/){target=_blank} is a monitoring and alerting toolkit that collects metrics from the nodes. By accessing the provided links, you can see the metrics of the nodes in a web interface.
+
+TODO: add a screenshot of the metrics page
+
+### Checking logs
+
+To check the nodes’ logs, you can use the provided command listed by the output as 'Log Cmd'. For instance, to check the logs of the `alice` node, you can open a new terminal and run the following command:
+
+```bash
+tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-SEzfCidQ1za4/alice.log
+```
+
+After running this command, you will see the logs of the `alice` node in real-time, which can be useful for debugging purposes. The logs of the `bob` and `collator01` nodes can be checked similarly.
+
+## Running a test
+
+To run a test against the spawned network, you can use the [Zombienet DSL](../testing.md) to define the test scenario. For example, you can create a file named `minimal-example-test.zndsl` with the following content:
+
+```toml
```

Comment:
Marking this item as solved since it will be resolved when those folders are merged into the main branch. It will be addressed at https://opslayer.atlassian.net/browse/DRP-40